### PR TITLE
reflect antismash record type spelling

### PR DESCRIPTION
**Pull request by nlouwen**
small misc changes: make sure that record type enum has same spelling as antismash (enables --record_type cand_cluster), fix use of --label flag and parsing of float param

### CODE REVIEW

File: big_scape/enums/genbank.py [hunk 1]
```diff
@@ -5,6 +5,6 @@
 
 class RECORD_TYPE(Enum):
     REGION = "region"
-    CANDIDATE_CLUSTER = "cand_cluster"
-    PROTO_CLUSTER = "proto_cluster"
+    CAND_CLUSTER = "cand_cluster"
+    PROTOCLUSTER = "protocluster"
```
Comment (adraismawur) [inline big_scape/enums/genbank.py:9]: I see what you're going for, but let's keep the variable naming according to the PEP8 style guide; for enums this is capitals and underscores.

CAND_CLUSTER is fine

**Comment by adraismawur** [comment]
This is always a thing with enums, but partly the point of the enums is that you don't really care about the actual values that you assign to them; they're more a naming tool so you can keep the code in a way that you expect.

If there are other locations where we still hard-code strings like `"proto_cluster"` we should replace those with enum values.

**Comment by adraismawur** [comment]
The rest looks fine!

**Review by adraismawur — CHANGES_REQUESTED** [review]

**Comment by nlouwen** [comment]
reverted to PROTO_CLUSTER, amended the validation so that "protocluster" is accepted from commandline

**Comment by nlouwen** [comment]
Adding on two more misc fixes:
- fix profiling validation check (was always false)
- fix progress bar logging (t.update takes an increment to add, not the total)
- fix indent in calculate_distances_mix
- added missing arg in calculate_distances_query

**Review by adraismawur — APPROVED** [review]
lgtm

**Pull request merged**